buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -678,6 +680,8 @@ class BeamModulePlugin implements Plugin<Project> {
         joda_time                                   : "joda-time:joda-time:2.10.10",
         jsonassert                                  : "org.skyscreamer:jsonassert:1.5.0",
         jsr305                                      : "com.google.code.findbugs:jsr305:$jsr305_version",
+        json_org                                    : "org.json:json:${json_org_version}",
```

Comment:
Thanks for the change! We already have a couple of json parser dependencies (e.g. jackson, gson, fastjson) throughout beam project. And there is some voice to reduce them (#24154). Could one (or some) of these libs satisfy the need of functionality here, or it is necessary to introduce these new deps?

Reply:
thanks for the observation @Abacn - let's keep this discussion in https://github.com/apache/beam/pull/24271 where the code is added? : )

Reply:
this PR is blocked by https://github.com/apache/beam/pull/24271